boringssl-fips/build/crypto/chacha/chacha-x86_64.S

```diff
@@ -0,0 +1,1632 @@
+# This file is generated from a similarly-named Perl script in the BoringSSL
```

Comment:
I think we just need the `*.a` files in build/ (but everything under include/). Can we clean these up a bit? Especially the 800k line one :-)